week04_approx_rl/homework_tf.ipynb

```diff
@@ -125,7 +185,7 @@
     "import gym\n",
     "# spawn game instance for tests\n",
     "env = gym.make(\"BreakoutNoFrameskip-v4\")  # create raw env\n",
-    "env = PreprocessAtari(env)\n",
+    "env = PreprocessAtariObs(env)\n",
```

Comment:
The wrapper above is still called `PreprocessAtari`. Please rename it or revert the change on this line.